### PR TITLE
docs(readme): update readme instructions for running app

### PR DESCRIPTION
**Pull request by seaerchin**
## Problem
app requires more setup to run. this isn't well documented so this pr adds the required steps

## Solution
1. delete unused commands
2. add set up steps 
3. combine required commands

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -112,7 +111,6 @@
     "lint-staged": "^11.1.2",
     "prettier": "2.2.1",
     "regenerator-runtime": "^0.13.9",
-    "serverless": "^3.22.0",
     "serverless-offline": "^11.0.1",
     "serverless-offline-lambda": "^1.0.6",
     "serverless-plugin-typescript": "^2.1.2",
```
Comment (seaerchin) [inline package.json:116]: @kishore03109 double checking - all the `serverless-xxx` stuff can be removed?

File: package-lock.json [hunk 2]
```diff
@@ -4,16 +4,6 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
-    "2-thenable": {
```
Comment (alexanderleegs) [inline package-lock.json:7]: Just to double check, are these dependencies all unused?

Comment (seaerchin) [reply package-lock.json:7]: not entirely sure how these came to be - if they're in our `deps` and we can't find a relevant import in our codebase, then i'm assuming they're unused (subpackages should specify their own deps)

File: package.json (hunk 1 above)
Comment (kishore03109) [reply package.json:116]: yup, can be removed!

**Review by kishore03109 — APPROVED** [review]

**Review by alexanderleegs — APPROVED** [review]

File: package-lock.json (hunk 2 above)
Comment (seaerchin) [reply package-lock.json]: removed all extra dpes

**Pull request merged**